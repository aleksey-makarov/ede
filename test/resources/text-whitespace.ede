{ "whitespace": "  A Word  " }
---
{{ whitespace | strip }}
{{ whitespace | stripStart }}
{{ whitespace | stripEnd }}
