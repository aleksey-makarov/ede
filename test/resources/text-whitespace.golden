A Word
A Word  
  A Word
